assets/src/scripts/utils.js

```diff
@@ -290,3 +290,20 @@ export const getNearestTime = function(data, time) {
     // Return the nearest data point and its index.
     return datum;
 };
+
+/*
+* When given an approval code, will return the text equivalent
+*  @param {String} approvalCode - Usually a letter such as 'A'
+*   @return {String} - an easy to understand text version of an approval code
+*/
+export const approvalCodeText = function(approvalCode) {
```

Comment:
Why put this here? It is only used in components/hydrograph/selectors/discrete-data? At this point, approval encoding is different for each kind of data (i.e. iv, dv, and gw) so I don't think it's generally usable.

Reply:
Gotcha. Yeah, I didn't know if there was cross over between the data types so thought it might be a candidate for reuse. WIll move.